test/fn/convert-forms.utils.js

```diff
@@ -16,23 +16,19 @@ module.exports = {
       // recursively copy forms and expected XML to temp directory, and create
       // tests dynamically
 
-      const targetDir = `${projectDir}/forms/${type}`;
+      const expectedDir = `${projectDir}/forms/${type}/expected`;
 
-      fs.recurseFiles(targetDir)
-        .filter(file => file.endsWith('.expected.xml'))
-        .forEach(expectedXml => {
+      fs.recurseFiles(expectedDir).forEach(expectedXml => {
 
-          const generatedXml = expectedXml.replace(/\.expected\.xml$/, '.xml');
-
-          it(`should generate ${generatedXml} as expected`, () => {
-
-            assert.ok(fs.exists(generatedXml), `Missing generated XML file: ${generatedXml}`);
-            assert.equal(fs.read(generatedXml), fs.read(expectedXml), `Content of ${generatedXml} was not as expected.`);
-
-          });
+        const generatedXml = expectedXml.replace(/\/expected/, '');
```

Comment:
Regex seems unrequired now. Can we replace `'/expected/'` with `'/'`?

Reply:
Done.